extensions/quit-application/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "quit-application",
+  "title": "Quit Application",
+  "description": "Quit apps directly from Raycast",
+  "icon": "quit-app.png",
+  "author": "mackopes",
+  "categories": [
+    "System",
+    "Productivity"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "Quit Application",
+      "subtitle": "System",
```

Comment:
Could you use another subtitle or remove it completely? It would be better to now mix up public extensions with our Raycast native System commands